public/api/indexRestaurants.json

```diff
@@ -0,0 +1,238 @@
+[
```

Comment:
In this file, it works to keep all of your data in one file that you can iterate over, but for better organization, you could pull each of these restaurants and their info into their own separate .json files like you have done with some, but not all. Then you can keep in your indexRestaurants.json file an array with each of the restaurants' ids. Ex:
```js
[
"ahli-babas-kabob",
"american-flatbread",
"august-first",
... etc
]
```

Then you can fetch the array of names to iterate over and from there fetch each of the json files. Yes, it may be a little bit annoying / tedious to implement, but it's good practice to keep your info separated like this to keep things organized.

Aside from organization, another reason for this is, say you had hundreds, or thousands of restaurants, you may not want to grab ALL of the data all at once, but say only 50 restaurants at a time. You can iterate over the first 50 restaurants and get all the info for those, and then if the user wanted to load more, they could hit load more and you can iterate through the next 50 restaurants and get that data. This prevents your site from taking the time to load so much data all at once but only get what you need.